pynumaflow/function/server.py

```diff
@@ -45,9 +46,15 @@ class UserDefinedFunctionServicer(udfunction_pb2_grpc.UserDefinedFunctionService
     >>> grpc_server.start()
     """
 
-    def __init__(self, map_handler: UDFMapCallable, sock_path=FUNCTION_SOCK_PATH):
+    def __init__(
```

Comment:
shouldn't we document the newly added arg?

Reply:
the new arg needs to be added in the docstring